lib/managers/files.js

```diff
@@ -575,6 +575,73 @@ Files.prototype.deletePermanently = function(fileID, callback) {
 	this.client.del(apiPath, null, this.client.defaultResponseHandler(callback));
 };
 
+
+ /**
+  * Enum of valid actions for locking and unlocking files
+  *
+  * @enum {LockType}
+  */
+Files.prototype.lockType = {
```

Comment:
We typically also expose these sorts of constants on the client itself, and the SDK object; see https://github.com/box/box-node-sdk/blob/master/lib/box-client.js#L356 and https://github.com/box/box-node-sdk/blob/master/lib/box-node-sdk.js#L118